docs/home/changelog/hotfixes-2-15.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Changelog Version 2.15
+summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch version.
+authors:
+    - Jason Novich
+date: 2024-Feb-25
+---
+
+The following is a list of the known and fixed issues for Run:ai V2.15.
+
+## Version 2.15.9 - February 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15296 | Fixed an issue where the `resources` field was deprecated when creating a project. Now use `nodePoolResources. |
+
+## Version 2.15.4 - January 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15026 | Fixed an issue where building workloads to send to the cluster were not built inside volumes for backwards compatibility. |
+| RUN-14907 | Fixed an issue where after an upgrade, the Analytics Dashboard is missing the time ranges from before the upgrade.|
+| RUN-14903 | Fixed an issue where the project status is audited creating long logs. |
+| RUN-14062 | Fixed an issue where the Running Workload per Type Graph box content is too big for the frame. |
+
+## Version 2.15.2 - February 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-14645 | Fixed an issue where a workload volume is mounted as read-only when created in the workspace training form. |
+| RUN-14635 | Fixed an issue where the Status updater was missing the correct permissions. |
+| RUN-14472 | Fixed an issue where template updates were not being applied to the workload. |
+| RUN-14434 | Fixed an issue where `runai_allocated_gpu_count_per_gpu` was multiplied by seven. |
+| RUN-14425 | Fixed an issue when calculating deserved quota metrics for project, nodepool, or cluster causing unlimited quotas to display -1. |
+| RUN-13401 | Fixed an issue where there was no accurate error information when a predicate fails (was `not found`). |
```

Comment:
what can a customer do with this info? Did you check if we have the term predicate documented??